QuadradiusR/scripts/player_list.gd

```diff
@@ -10,17 +10,17 @@ onready var new_player_joined_sfx = $NewPlayerJoinedSFX
 onready var player_left_sfx = $PlayerLeftSFX
 
 const player_template = preload("res://prefabs/player_list_row.tscn")
+var active_invites = {}  # uuid -> game invite
```

Comment:
This variable can be called `uuid_to_active_invites` which makes it more clear what sort of mapping is this 😃